tests/nemo_text_processing/en/data_text_normalization/test_cases_telephone.txt

```diff
@@ -1,12 +1,15 @@
-+1 123-123-5678~one, one two three, one two three, five six seven eight
++1 123-123-5678~plus one, one two three, one two three, five six seven eight
 123-123-5678~one two three, one two three, five six seven eight
-+1-123-123-5678~one, one two three, one two three, five six seven eight
-+1 (123)-123-5678~one, one two three, one two three, five six seven eight
-+1(123) 123-5678~one, one two three, one two three, five six seven eight
++1-123-123-5678~plus one, one two three, one two three, five six seven eight
++1 (123)-123-5678~plus one, one two three, one two three, five six seven eight
++1(123) 123-5678~plus one, one two three, one two three, five six seven eight
 (123)-123-5678~one two three, one two three, five six seven eight
 (123) 123-5678~one two three, one two three, five six seven eight
 123-123-5678-1111~one two three, one two three, five six seven eight, one one one one
 123-123-5678-1~one two three, one two three, five six seven eight, one
 1-800-GO-U-HAUL~one, eight hundred, GO U HAUL
 1-800-GO-2-HAUL~one, eight hundred, GO two, HAUL
-123.123.0.40~one two three dot one two three dot o dot four o
+123.123.0.40~one two three dot one two three dot zero dot four zero
+call me at +1 123-123-5678~call me at plus one, one two three, one two three, five six seven eight
+call me at 123-123-5678~call me at , one two three, one two three, five six seven eight
+IP address is 123.123.0.40~IP address is , one two three dot one two three dot zero dot four zero
```

Comment:
why is a comma added in the last 2 test cases?

Reply:
since it is used after country code and i can only use prompt in country code. After country code there is a comma